--- a/thetvdb.py
+++ b/thetvdb.py
@@ -350,7 +350,10 @@
 		if notext:
 			return notext["image"]
 
-		return None
+		english = next((x for x in items if x["language"] == "eng"), None)
+		if english:
+			return english["image"]
+		return items[0]["image"] if items else None
 
 	def _process_remote_id(
 		self, ids: list[dict[str, Any]] | None
@@ -423,7 +426,8 @@
 				tags=[],
 				poster=data.get("image")
 				if lang == "eng"
-				else self._pick_image(show["artworks"], kind, "posters", lang),
+				else self._pick_image(show["artworks"], kind, "posters", lang)
+				or data.get("image"),
 				thumbnail=self._pick_image(show["artworks"], kind, "backgrounds", lang),
 				banner=self._pick_image(show["artworks"], kind, "banners", lang),
 				logo=self._pick_image(show["artworks"], kind, "icons", lang),
